app/views/tripfamilies/_tripfamily.html.erb

```diff
@@ -0,0 +1,7 @@
+<tr>
```

Comment:
The name of this file should be `_trip_family.html.erb`

Reply:
Also the folder name is incorrect...

Reply:
Done in sha `e2a4c4e`